libkbfs/mdserver_remote.go

```diff
@@ -71,6 +71,8 @@ type MDServerRemote struct {
 	serverOffsetMu    sync.RWMutex
 	serverOffsetKnown bool
 	serverOffset      time.Duration
+
+	reconnectBackoff *CancelableRandomBackoff
```

Comment:
`*CancelableRandomBackoff` is a type (pointer to `CancelableRandomBackoff`) and in Go this is traditionally written with no space.

But thanks for reviewing! 😊